utilMenu.java

```diff
@@ -41,8 +41,8 @@ public static void stareaContului(String ID) {
     //metoda care ajuta metoda-mama optiuni
     public static void modificareOptiune(String ID, int pret, String Optiunea, String MINGB, int cantitatea) {
         int x;
-        if (Objects.equals(MINGB, "MIN")) x = 2;
-        else x = 1;
+        if (Objects.equals(MINGB, "GB")) x = 1;
+        else x = 2;
 
```

Comment:
eu la altceva ma refeream.
Uite tu creezi x fara valoare
verifici o conditie, if true x=1
daca not true x=2


in loc de asta e mai eficient sa creezi x=2 de baza
daca conditia este egal true x=1